set.mm

```diff
@@ -528647,6 +528698,20 @@ every nonzero prime ideal contains a prime element (this is a
         JKLZUEUFMZUDUGUHNZUDBOMCOMUDUIPBCQBCRBCSTUAUBUC $.
     $}
 
+    ${
+      sdrgfldext.b $e |- B = ( Base ` E ) $.
+      sdrgfldext.e $e |- ( ph -> E e. Field ) $.
+      sdrgfldext.f $e |- ( ph -> F e. ( SubDRing ` E ) ) $.
+      $( In a field, sub-division-rings are field extensions.  (Contributed by
```

Comment:
A nitpick, but a ring or a field cannot be a field extension, since the latter is a pair of fields. Maybe it's better to write something like "A field and its sub-division-ring form a field extension."